lib/collection/src/operations/mod.rs

```diff
@@ -180,6 +180,40 @@ impl CollectionUpdateOperations {
             Self::FieldIndexOperation(_) => (),
         }
     }
+
+    pub fn filter(&self) -> Option<&Filter> {
```

Comment:
We already have this https://github.com/qdrant/qdrant/blob/c95b2448d9231ac4488798af30d718965766d0d8/lib/collection/src/operations/operation_effect.rs#L28

Which might be both: a better fit (cause also includes points, which we need to count as individual updates as well), and require some upgrade, cause we might not want to clone `Filter`

Reply:
This operation effect area is only used in ProxyShard, but ProxyShard itself is not used anywhere, so, if needed, we can change some stuff in the signatures if needed